EveripediaNetwork/ep-api
# Wiki Validation Service

# Wiki Validation Service

_A service that validates checks for the wiki content we get from IPFS data on the Indexer Service_

## How should this be tested?

1. Have a IPFS hash run through the Indexer Service
2. Check out if IPFS has been validated correctly
3. If the service worked fine, it's good to go

## Notes or observations

I have had some hard time with typescript on this one and I might have not chosen the best variables names on the reducer that run the checks functions - if you have any suggestion or way to make the code better or readable, awesomely welcome! 

other than that, I think I've gone through close to the best approach we might have here. I didn't want a lot of conditions over the code, wanted something simple to read and also to add another check or remove another check we might have without having a dirty chained condition code. I actually would like to have a second opinion on that and see if I did ok

## Linked issues

closes #00, closes #001, closes https://github.com/EveripediaNetwork/issues/issues/238
